views/group/new.pug

```diff
@@ -19,15 +19,25 @@ block content
       div
         label(class='text-lg font-bold' for='tags') Címkék (8)
         input(type="tags", name="tags", id="tags", class="w-full px-3 py-2 rounded form-element")
-      div
-        label(for="room" class='text-lg font-bold') Szint
+      div(class="flex space-x-2")
+        button(type="button", id="floorBtn", onclick="selectMeetingPlace('floor')", class='self-start btn btn-meeting-default animate-hover' class=`${(!isEditing || roomId) ? 'btn-meeting-selected' : ''}`) Szint
+        button(type="button", id="linkBtn", onclick="selectMeetingPlace('link')", class='self-start btn btn-meeting-default animate-hover' class=`${(isEditing && link) ? 'btn-meeting-selected' : ''}`) Link
+        button(type="button", id="otherBtn", onclick="selectMeetingPlace('other')", class='self-start btn btn-meeting-default animate-hover' class=`${(isEditing && place) ? 'btn-meeting-selected' : ''}`) Egyéb
```

Comment:
The purpose of these buttons might not be clear to new users.
Maybe the buttons could have a label like 'Helyszín', and then the button texts could say 'Tanulószoba', 'Online' and 'Egyéb'.

Reply:
If we want to describe the purpose of these buttons further, we can put a small `?` on them and tooltips on them, but I think it's alright right now 👌